src/microbe_stage/Microbe.cs

```diff
@@ -1420,25 +1420,29 @@ internal void SuccessfulKill()
 
     private void OnIGotAddedToColony()
     {
-        var oldRotation = Rotation;
-        var vectorToParent = GlobalTransform.origin - ColonyParent.GlobalTransform.origin;
-        ChangeNodeParent(ColonyParent);
-
         State = MicrobeState.Normal;
 
-        var vectorToParentRotated = vectorToParent.Rotated(Vector3.Down, Rotation.y);
-        var vectorToMembrane = Membrane.GetExternalOrganelle(vectorToParentRotated.x, vectorToParentRotated.y);
+        if (ColonyParent != null)
```

Comment:
I think an early return would be better. Something like
```cs
if (ColonyParent == null)
{
    UnreadyToReproduce();
    return;
}
// code...
```
would be more readable